.github/workflows/requirements.yml

```diff
@@ -26,7 +26,7 @@ env:
   MLFLOW_HOME: /home/runner/work/mlflow/mlflow
   MLFLOW_CONDA_HOME: /usr/share/miniconda
   SPARK_LOCAL_IP: localhost
-  PYTHON_VERSION: "3.8"
+  PYTHON_VERSION: "3.10" # mininum supported version + 1
```

Comment:
To run tests newer versions of package and detect issues early.

Reply:
Don't we need to test it against current supported version?

Reply:
We do in `master.yml` :)